tests/integration/fixtures.py

```diff
@@ -255,6 +256,23 @@ def _assert_query_gives_same_result(query, sort_columns=None, **kwargs):
     return _assert_query_gives_same_result
 
 
+def gpu_cluster():
+    if LocalCUDACluster is None:
+        pytest.skip("dask_cuda not installed")
+        return None
+
+    cluster = LocalCUDACluster(protocol="tcp")
+    yield cluster
+    cluster.close()
+
+
+def gpu_client(gpu_cluster):
+    if gpu_cluster:
+        client = Client(gpu_cluster)
+        yield client
+        client.close()
```

Comment:
No reason. Just felt the right thing to do in case when we  want to use an external scheduler later on .  `Dask-sql` has some separate tests with external cpu scheduler.   [Link code](https://github.com/dask-contrib/dask-sql/blob/61a20b78877d96e395b359e11c9a6a9c5c554cdb/.github/workflows/test.yml#L166-L179)

Then `gpu_client` can just be used accordingly.